lib/mumuki/classroom/models/assignment.rb

```diff
@@ -72,6 +70,24 @@ def add_message_to_submission!(message, sid)
     submission
   end
 
+  #TODO use JSON Builder
```

Comment:
Perhaps I myself added this comment, but I think JSON Builder make more sense at controller level, not domain model.

Reply:
Removed comment!